package.json

```diff
@@ -8,10 +8,10 @@
     "node": ">=8.10.0"
   },
   "repository": "googleapis/nodejs-automl",
-  "main": "src/index.js",
+  "main": "build/src/index.js",
   "files": [
-    "protos",
-    "src"
+    "build/protos",
```

Comment:
Interesting.  @alexander-fenster in other places, we seem to import from just `"protos"`.  What's the difference?

Reply:
Are you talking about `.jsdoc.js` or `package.json`?  In `package.json`, it's always `build/protos` AFAIK.